config/default/webform.webform.feedback_form.yml

```diff
@@ -63,11 +63,9 @@ elements: |-
       '#description': |-
         Please do not enter any personal information in this space
         <div id="extension-host">&nbsp;</div>
-  have_you_liked_our_new_intranet_site_:
-    '#type': textfield
-    '#title': 'Have you liked our new Intranet site? '
-    '#description': 'Please give your feed back&nbsp;'
-    '#required': true
+    do_you_love_this_website_so_much_:
+      '#type': checkbox
+      '#title': 'Do you love this website so much?'
```

Comment:
Do you love this website so much?

Is this an official change?

Reply:
Sorry, it's not simple to keep the webform changes out of the config export, but these files need to be reverted.

Reply:
Nah I've removed it like 15 times 😆